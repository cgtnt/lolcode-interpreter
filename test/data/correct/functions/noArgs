HAI

HOW IZ I ADD
  FOUND YR "PRIUNTABLE"
IF U SAY SO

VISIBLE I IZ ADD MKAY

KTHXBYE
