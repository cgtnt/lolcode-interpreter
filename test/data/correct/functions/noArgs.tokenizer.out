@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_FUNC_BEGIN:HOW IZ I-@3:T_T_IDENTIFIER:ADD-@4:T_COMMAND_TERMINATOR:
-@4:T_RETURN_VAL:FOUND YR-@4:T_T_STRING:"PRIUNTABLE"-@5:T_COMMAND_TERMINATOR:
-@5:T_FUNC_END:IF U SAY SO-@7:T_COMMAND_TERMINATOR:
-@7:T_WRITE_STDOUT:VISIBLE-@7:T_FUNC_CALL:I IZ-@7:T_T_IDENTIFIER:ADD-@7:T_END_INF:MKAY-@9:T_COMMAND_TERMINATOR:
-@9:T_END:KTHXBYE-@10:T_COMMAND_TERMINATOR:
-@10:T_EOF:
